.travis.yml

```diff
@@ -1,11 +1,10 @@
 sudo: false
 language: node_js
 node_js:
-    - "4"
-    - "5"
-    - "6"
-    - "7"
     - "8"
```

Comment:
We're not using 8 and never will. Is 8 required?

Reply:
No. I was using it, but I always got segfaults rather than descriptive error messages, thus I am not mostly working with 12. I will remove it.